src/Runner.cs

```diff
@@ -18,7 +18,6 @@ public class Runner
         protected static Dictionary<WatcherChangeTypes, Change> Changes = new Dictionary<WatcherChangeTypes, Change>();
 
         private List<Thread> _threads = new List<Thread>();
-        private bool _stopMonitoring = false;
```

Comment:
I'm not 100% sure if this was redundant, but it always coincided with `_stopMonitoringEvent.Set()`, so I thought I might try it.